src/timesync/timesyncd.c

```diff
@@ -208,6 +208,10 @@ static int run(int argc, char *argv[]) {
         if (r < 0)
                 return r;
 
+        r = bus_manager_emit_ntp_server_changed(m);
+        if (r < 0)
+                return r;
```

Comment:
why call this here? if the mask is zero anyway (and it is) there's no reason to call this

Reply:
yes, remove it